KomaMRIBase/src/datatypes/Phantom.jl

```diff
@@ -612,5 +549,107 @@ function check_phantom_arguments(nd, ss, us)
+    T2 =
+        (class .== 23) * tissue_properties["CSF"][2] .+ #CSF
+        (class .== 46) * tissue_properties["GM"][2] .+ #GM
+        (class .== 70) * tissue_properties["WM"][2] .+ #WM
+        (class .== 93) * tissue_properties["FAT1"][2] .+ #FAT1
+        (class .== 116) * tissue_properties["MUSCLE"][2] .+ #MUSCLE
+        (class .== 139) * tissue_properties["SKIN/MUSCLE"][2] .+ #SKIN/MUSCLE
+        (class .== 162) * tissue_properties["SKULL"][2] .+ #SKULL
+        (class .== 185) * tissue_properties["VESSELS"][2] .+ #VESSELS
+        (class .== 209) * tissue_properties["FAT2"][2] .+ #FAT2
+        (class .== 232) * tissue_properties["DURA"][2] .+ #DURA
+        (class .== 255) * tissue_properties["MARROW"][2] #MARROW
+    T2s =
+        (class .== 23) * tissue_properties["CSF"][3] .+ #CSF
+        (class .== 46) * tissue_properties["GM"][3] .+ #GM
+        (class .== 70) * tissue_properties["WM"][3] .+ #WM
+        (class .== 93) * tissue_properties["FAT1"][3] .+ #FAT1
+        (class .== 116) * tissue_properties["MUSCLE"][3] .+ #MUSCLE
+        (class .== 139) * tissue_properties["SKIN/MUSCLE"][3] .+ #SKIN/MUSCLE
+        (class .== 162) * tissue_properties["SKULL"][3] .+ #SKULL
+        (class .== 185) * tissue_properties["VESSELS"][3] .+ #VESSELS
+        (class .== 209) * tissue_properties["FAT2"][3] .+ #FAT2
+        (class .== 232) * tissue_properties["DURA"][3] .+ #DURA
+        (class .== 255) * tissue_properties["MARROW"][3] #MARROW
+
+    ρ =
+        (class .== 23) * tissue_properties["CSF"][4] .+ #CSF
+        (class .== 46) * tissue_properties["GM"][4] .+ #GM
+        (class .== 70) * tissue_properties["WM"][4] .+ #WM
+        (class .== 93) * tissue_properties["FAT1"][4] .+ #FAT1
+        (class .== 116) * tissue_properties["MUSCLE"][4] .+ #MUSCLE
+        (class .== 139) * tissue_properties["SKIN/MUSCLE"][4] .+ #SKIN/MUSCLE
+        (class .== 162) * tissue_properties["SKULL"][4] .+ #SKULL
+        (class .== 185) * tissue_properties["VESSELS"][4] .+ #VESSELS
+        (class .== 209) * tissue_properties["FAT2"][4] .+ #FAT2
+        (class .== 232) * tissue_properties["DURA"][4] .+ #DURA
+        (class .== 255) * tissue_properties["MARROW"][4] #MARROW
+
+    Δw =
+        (class .== 23) * tissue_properties["CSF"][5] .+ #CSF
+        (class .== 46) * tissue_properties["GM"][5] .+ #GM
+        (class .== 70) * tissue_properties["WM"][5] .+ #WM
+        (class .== 93) * tissue_properties["FAT1"][5] .+ #FAT1
+        (class .== 116) * tissue_properties["MUSCLE"][5] .+ #MUSCLE
+        (class .== 139) * tissue_properties["SKIN/MUSCLE"][5] .+ #SKIN/MUSCLE
+        (class .== 162) * tissue_properties["SKULL"][5] .+ #SKULL
+        (class .== 185) * tissue_properties["VESSELS"][5] .+ #VESSELS
+        (class .== 209) * tissue_properties["FAT2"][5] .+ #FAT2
+        (class .== 232) * tissue_properties["DURA"][5] .+ #DURA
+        (class .== 255) * tissue_properties["MARROW"][5] #MARROW
+    return T1, T2, T2s, ρ, Δw
+end
```

Comment:
This could be done in a for loop, so it outputs a Tuple, and to avoid writing the same thing 5 times only changing a number.